src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background-color: $white;
+  border: 1px solid $light-grey;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+  font-family: $roboto, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-family: $roboto, sans-serif;
+    color: $black;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    margin-bottom: 4px;
+    margin-top: 40px;
+
+    &:hover {
+      color: $dark-blue;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $grey;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__review {
+    font-size: 10px;
+    color: $black;
+    line-height: 14px;
+    text-align: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $grey;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $black;
+    margin-bottom: 16px;
+  }
+
+  &__button {
+    text-align: center;
+  }
+
+  &__link-buy {
+    box-sizing: border-box;
+    display: block;
+    font-family: $roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 38px;
+    color: $white;
+    background-color: $blue;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $blue;
+    text-transform: uppercase;
+  }
+
+  &__link-buy:hover {
+    background-color: $white;
+    color: $blue;
+    transition: all 0.2s;
```

Comment:
Transition should be added to the default state of the element, not hover

Reply:
done